support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -10,6 +10,7 @@ export const pageUrlRegexes = {
 		allLandingPagesAndThankyouPages:
 			'/checkout|one-time-checkout|contribute|thankyou|thank-you(/.*)?$',
 		usLandingPageOnly: '/us/contribute$',
+		genericCheckoutOnly: '/checkout|thank-you(/.*)?$',
```

Comment:
I have a feeling this will also pick up the existing one-time checkout at `/contribute/checkout`, updating to `'(uk|ca|eu|nz|int)/checkout|thank-you(/.*)?$'` should fix that? I believe we have unit tests in `abtestTest.ts` for validating these regexes, could be handy adding some to confirm.

Reply:
Great spot!